source/tutorials/aggregation.txt

```diff
@@ -0,0 +1,118 @@
+.. _scala-aggregation:
+
+=====================
+Aggregation Framework
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, create insights, change data
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+The aggregation pipeline is a framework for data aggregation,
+modeled on the concept of data processing pipelines.
+
+To learn more about aggregation, see :manual:`Aggregation Pipeline
+</core/aggregation-pipeline/>` in the Server manual.
+
+Prerequisites
+-------------
+
+.. include:: /includes/prereq-restaurants.rst
+
+.. code-block:: scala
+
+   import org.mongodb.scala._
+
+   import org.mongodb.scala.model.Aggregates._
+   import org.mongodb.scala.model.Accumulators._
+   import org.mongodb.scala.model.Filters._
+   import org.mongodb.scala.model.Projections._
+
+.. include:: /includes/obs-note.rst
+
+Connect to a MongoDB Deployment
+-------------------------------
+
+.. include:: /includes/connect-section.rst
+
+Perform Aggregation
+-------------------
+
+To perform aggregation, pass a list of aggregation stages to the
+``MongoCollection.aggregate()`` method. The driver provides the
+``Aggregates`` helper class that contains builders for aggregation
+stages.
+
+In this example, the aggregation pipeline performs the
+following tasks:
+
+- Uses a ``$match`` stage to filter for documents in which the
+  ``categories`` array field contains the element ``"Bakery"``. The example
```

Comment:
Question:
This mentions that it uses the "Aggregates.match()" method, but I don't see that anywhere in the example. Perhaps it should be Aggregates.filter?